src/main/java/com/meetup/server/event/dto/response/RouteResponse.java

```diff
@@ -40,6 +40,19 @@ public static RouteResponse of(StartPoint startPoint,
                                    User user,
                                    OdsayTransitRouteSearchResponse transitResponse,
                                    KakaoMobilityResponse drivingResponse) {
+
+        List<DrivingRouteResponse> drivingRouteResponseList = DrivingRouteResponse.from(drivingResponse);
+
+        int driveTime = 0;
+        if (drivingRouteResponseList != null && !drivingRouteResponseList.isEmpty() && drivingRouteResponseList.get(0) != null) {
+            driveTime = drivingRouteResponseList.get(0).duration();
```

Comment:
```suggestion
            driveTime = drivingRouteResponseList.getFirst().duration();
```
JDK21에서 나온 getFirst()를 사용해도 좋을 거 같아요!